http-client/src/test/java/io/airlift/http/client/jetty/TestJettyHttpClientHttpProxyAuthN.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.http.client.jetty;
+
+public class TestJettyHttpClientHttpProxyAuthN
+        extends AbstractHttpClientTestHttpProxyAuthN
```

Comment:
Do we need abstract class? Merge these classes together

Reply:
If we merge them, the TestJettyHttpClientHttpProxyAuthN will need to have two constructor, one for init the class, and the second with parameter for the extension in derived class. However, the Junit framework demands single constructor for the test class.